about.html

```diff
@@ -0,0 +1,109 @@
+                </p>
+
+            </div>
+            <div class="conference-location">
+                <div class="contact-massage lato-font magin-block-cancel">For any inquiries, you can contact us via:</div>
+                <div class="email cocogoose-font magin-block-cancel">hammaagazarok@gmail.com</div>
+            </div>
+        </section>
+        <section id="logo" class="committees">
+            <h1 class="main-header-text color1">Fourth International Conference <br>
+                on Science and Technology</h1>
+            <div class="committees-line"></div>
+            <div class="logo-list-container">
+                <ul class="logo-list magin-block-cancel">
+                <li class="remove-decoration">
+                    <p>Under the slogan "for Sustainable Development"</p>
+                </li> 
+                <li class="remove-decoration">
+                    <img src="assets/logo.png" alt="ict4 logo">
+                </li>
+                   
+                </ul>
+            </div>
+        </section>
+        <div class="line"></div>
+        <section id="history" class="history">
+            <h2 class="main-header-text color1">See The Past ICST Conferences</h2>
+            <div class="committees-line"></div>
+            <div class="history-list-container">
+                <ul class="history-list magin-block-cancel">
+                    <li class="remove-decoration" >
+                        <p>Take a look at the last two ICST Conferences that took place in Brack and Sebha Cities</p>
+                    </li>
+                    <li class="remove-decoration" >
+                        <div class="history-images">
+                            <img src="assets/2019.png" alt="ict4 logo">
+                            <img src="assets/2020.png" alt="ict4 logo">
+                        </div>
+                    </li>
+                    
+                    
+                </ul>
+            </div>
+        </section>
+        <div class="line"></div>
+        <footer class="footer-about">
+            <div class="footer-content-container">
+                <ul class="footer-content">
+                    <li class="footer-item">
+                        <img src="assets/footer-right.png" alt="copyright">
+                        <div>
+                            <p class="color1 lato-font" >Fourth International Conference on Science and Technology</p>
+                            <p class="copyright_subtitle color1 lato-font"> </p>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+        </footer>
```

Comment:
- [ ] As requested by the previous reviewer kindly make the `About` page have a dark background on the footer and the `text` have white color. See design below
### Behance design
![design](https://user-images.githubusercontent.com/81727988/175300306-7d7f2a8c-f036-49d2-9000-e8f471e4362d.png)
### Your design
![his](https://user-images.githubusercontent.com/81727988/175300338-152d57a2-9e02-4fb1-9ff5-7402f0e2235a.png)